symtabAPI/src/Archive-elf.C

```diff
@@ -32,7 +32,7 @@
 
 #include "symtabAPI/h/Symtab.h"
 #include "symtabAPI/h/Archive.h"
-#include "symtabAPI/src/Object.h"
+#include "symtabAPI/src/Object-elf.h"
```

Comment:
I don't see any use of the class Object-elf in this class.
Should it be Elf_X.h or maybe other relevant header like elf.h instead?

Reply:
Agreed. I updated it to "Elf_X.h" and fixed another missing transitive include.